flink-python/pyflink/datastream/stream_execution_environment.py

```diff
@@ -243,7 +244,7 @@ def get_state_backend(self):
         j_state_backend = self._j_stream_execution_environment.getStateBackend()
         return _from_j_state_backend(j_state_backend)
 
-    def set_state_backend(self, state_backend):
+    def set_state_backend(self, state_backend: StateBackend):
         """
         Sets the state backend that describes how to store and checkpoint operator state. It
```

Comment:
add type hint for set_restart_strategy/get_restart_strategyj, set_stream_time_characteristic/get_stream_time_characteristic?